rfcs/select-component.md

```diff
@@ -153,6 +153,8 @@ export default () => {
   Whether the control should be in an error state. Default is `false`
 * `multiple: ?boolean`:
   Sets if multiple choices are allowed in Select component. Default is `false`
+* `maxHeight: ?string`:
```

Comment:
We should also delete `rows` (line 146) from the API doc here

Reply:
Since this property is exposed on the select, how about we name it something like `maxMenuHeight` or `maxDropdownHeight` so there is no confusion that it can be related to the select's height?

Reply:
I'd go with `maxMenuHeight` as https://react-select.com/props

Reply:
makes sense